libs/order/src/models/order-item.ts

```diff
@@ -0,0 +1,25 @@
+import { DaffProductImage } from '@daffodil/product';
+
+export interface DaffOrderItem {
+  item_id: number;
+  image: DaffProductImage;
+  quote_id: number;
```

Comment:
should be `order_id: string | number`

Reply:
I'll make the order IDs be `DaffOrder['id']`

Reply:
changed